template/tsconfig.json

```diff
@@ -36,8 +36,11 @@
         /* Module Resolution Options */
         "moduleResolution": "node",               /* Specify module resolution strategy: 'node' (Node.js) or 'classic' (TypeScript pre-1.6). */
         "esModuleInterop": true,                  /* */
-        // "baseUrl": "./",                       /* Base directory to resolve non-absolute module names. */
-        // "paths": {},                           /* A series of entries which re-map imports to lookup locations relative to the 'baseUrl'. */
+        "baseUrl": "./",                          /* Base directory to resolve non-absolute module names. */
+        "paths": {                                /* A series of entries which re-map imports to lookup locations relative to the 'baseUrl'. */
+          "react": ["./node_modules/preact/compat"],
+          "react-dom": ["./node_modules/preact/compat"]
```

Comment:
This wont work in monorepos, you could do something along the lines of what Next.js does: https://github.com/vercel/next.js/blob/ffa378901f9b046021b3cc916e882e7afc99e3af/examples/with-typescript/next-env.d.ts
https://github.com/vercel/next.js/blob/ffa378901f9b046021b3cc916e882e7afc99e3af/examples/with-typescript/tsconfig.json#L17

Reply:
Well, this project template isn't a monorepo. If users want to put it in one they may need to make adjustments.

Setting the paths like that shouldn't cause issues even if the paths cannot be resolved, AFAIK.